### PR TITLE
chore: Update APK naming convention

### LINKED ISSUE #1764: Update APK generation and follow name conventions
Please reduce the number of APKs generated and change the generation of the APK file as follows:

- eventyay-organizer-dev-debug.apk
- eventyay-organizer-dev-debug-fdroid.apk
- eventyay-organizer-master-debug.apk
- eventyay-organizer-master-debug-fdroid.apk
- eventyay-organizer-master-unsigned.apk
- eventyay-organizer-master-unsigned-fdroid.apk

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1764

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Wait till #1770

**Comment by ShridharGoel** [comment]
@iamareebjamal Since #1770 is merged, any changes required in this?

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -26,17 +26,21 @@ fi
...
-    done
-fi
+for file in app*; do
 
-if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
-    for file in app*; do
-          mv $file eventyay-organizer-dev-${file%%}
-    done
-fi
+    if [ ${file} =~ "unaligned" ]; then
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: unsigned and unaligned files are not generated now

Comment (ShridharGoel) [reply scripts/update-apk.sh]: Updated.

**Comment by ShridharGoel** [comment]
> https://github.com/fossasia/open-event-organizer-android/blob/fe47981075f71b2fb285af9cbb976e7b338d58bc/scripts/update-apk.sh#L67
> 
> 
> I think `eventyay-organizer-master-app.aab` should replace with `eventyay-organizer-master.aab`.

@iamareebjamal Please confirm.

**Comment by iamareebjamal** [comment]
yes

**Comment by ShridharGoel** [comment]
Updated.

**Pull request merged**